src/Platform/Microsoft.Testing.Extensions.VSTestBridge/ObjectModel/ObjectModelConverters.cs

```diff
@@ -211,7 +211,7 @@ private static void AddOutcome(this TestNode testNode, TestResult testResult)
             // It seems that NUnit inconclusive tests are reported as None which should be considered as Skipped.
             case TestOutcome.None:
             case TestOutcome.Skipped:
-                testNode.Properties.Add(SkippedTestNodeStateProperty.CachedInstance);
+                testNode.Properties.Add(testResult.ErrorMessage is null ? SkippedTestNodeStateProperty.CachedInstance : new SkippedTestNodeStateProperty(testResult.ErrorMessage));
```

Comment:
Nit: let's split the ternary

```suggestion
                testNode.Properties.Add(testResult.ErrorMessage is null 
                    ? SkippedTestNodeStateProperty.CachedInstance
                    : new SkippedTestNodeStateProperty(testResult.ErrorMessage));
```